learn/react-native/wavepulse.md

```diff
@@ -0,0 +1,100 @@
+
+WavePulse is particularly useful for maintaining high-performance applications, ensuring stability, and providing end-users with a seamless experience. It enhances the process of debugging by allowing you to debug the APK or IPA file of an application without the need of using or sharing any confidential information like source code or excess information.
+
+To access WavePulse, [click here](#https://apps.wavemakeronline.com/wavepulse/client).
+
+### Key Features
+
+- **Easy Debug using IPA and APK**: Uses IPA or APK file to debug rather than using the React Native Zip file of an application.
+- **Quick and Simple Setup**: Simplifies the process of setup as it requires only one tool, WavePulse and few steps to get started with it.
+- **Debugging on Real Devices**: Debugging can be done on any device given WavePulse is enabled in the application to be debugged.
+- **Importing and Exporting Debugging Session**: Debugging session can be easily exported or imported using WavePulse without having to share any irrelevant or confidential information.
+- **Viewing WaveMaker Specific Data**: WaveMaker specific component and other details can be easily viewed. User can find every detail at one place, like:
+  - Console logs
+  - Network
+  - Element Tree with Properties and Styles
+  - Timeline of various events
+  - Storage
+  - Info
+
+## WavePulse Debug Architecture 
+
+Debugging is the process of identifying, analyzing, and resolving issues in an application. This step makes sure that the application functions as expected. WavePulse allows you to debug the APK or IPA file of an application on a mobile device where the application details required for debugging can be viewed in the web browser.
+
+![Debugging Architecture](/learn/assets/wavepulse-architecture.png)
+
+The debugging architecture using WavePulse includes the following steps.
+
+**Configuring APK/IPA File in WaveMaker Studio**
+
+1. In WaveMaker studio, [enable WavePulse](#enable-wavepulse-in-application) 
+2. [Generate APK or IPA file](#generate-apkipa-file-using-react-native-zip-file)
+
+**Testing & Debugging in WavePulse**
+
+3. Next for testing and debugging, open WavePulse application.
+4. In mobile device, download and install the APK or IPA file.
+
+**Connection between Web Browser and Mobile Device**
+
+5. Then establish [connection between WavePulse web application and mobile device by providing application ID](#connection-between-web-browser-and-mobile-device).
+6. Using mobile device scanner, scan QR code to run the application.
+7. Application is launched.
+8. Now, the debug session of the application can be viewed in WavePulse.
+
+## Configuring APK/IPA File in WaveMaker Studio
+
+An application can be debugged using the APK or IPA file. To get the APK or IPA that can be debugged using the WavePulse,
+
+- Enable the WavePulse in the application created in the WaveMaker studio.
```

Comment:
Enable logs also, if one wants to see the console logs.

Add a warning note saying, WaveMaker recommends WavePulse only for pre-release builds. In case of release builds, WaveMaker recommends to disable WavePulse.